sympy/concrete/tests/test_sums_products.py

```diff
@@ -1317,6 +1317,11 @@ def test_issue_14313():
     assert Sum(S.Half**floor(n/2), (n, 1, oo)).is_convergent()
 
 
+@XFAIL
+def test_issue_16735():
+    assert Sum(5**n/gamma(n+1), (n, 1, oo)).is_convergent() is True
```

Comment:
This can be `S.true` rather than `True`.

Reply:
I see that S.true issue. When it is True, the test fails and hence I added `@XFAIL`

Reply:
Please review the changes now.